framework/src/problems/SubProblem.C

```diff
@@ -1216,11 +1222,13 @@ SubProblem::jacobianSetup()
 void
 SubProblem::initialSetup()
 {
-  if (_output_functors)
+  if (_show_functors)
   {
     showFunctors();
     showFunctorRequestors();
   }
+  if (_show_chain_control_data)
+    _console << _app.getChainControlDataSystem().outputChainControlMap();
```

Comment:
```suggestion
    _console << _app.getChainControlDataSystem().outputChainControlMap() << std::flush;
```
Or don't flush above. Let's be consistent

Reply:
will update